### DISCUSSION TITLE
List doesn't update after creating Issue

### DISCUSSION THREAD

**Discussion by JVimes** [community/community#40972] [Projects and Issues]
After I add an issue, the issues list is visible but doesn't show the new issue unless I manually refresh by pulling down. The new issue should appear automatically. 

GitHub for Android version 1.91.0
Android 13 (Nov. 5, 2022)
Device: Pixel 4A

**Comment by ossanna16** [comment]
Hi there @JVimes and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
